spec/features/flower_shops/index_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe "flower shops index page", type: :feature do
+  before :each do
+    @mikes_flowers = FlowerShop.create!( name:"Mikes Flower shop",
+                                        rating: 3,
+                                        same_day_delivery: false,
+                                      )
+    @floral_designs = FlowerShop.create!( name:"Floral Designs",
+                                        rating: 4,
+                                        same_day_delivery: true,
+                                  )
+    @flower_depot = FlowerShop.create!( name:"Flower Depot",
+                                        rating: 5,
+                                        same_day_delivery: true,
+                                  )
+  end
+
+  it "can see the name of all the flower shops" do
+    visit "/flower_shops"
+
+    expect(page).to have_content(@mikes_flowers.name)
+    expect(page).to have_content(@floral_designs.name)
+    expect(page).to have_content(@flower_depot.name)
+  end
+end
```

Comment:
Eventually you'll want a test here that tests whether these link to their show pages

Reply:
```rb
it 'links to each garden show page' do
    visit '/gardens'

    click_on 'North Boulder Community Garden'

    expect(current_path).to eq("/gardens/#{@north_boulder.id}")
  end
```